pkg/virt-api/webhooks/validating-webhook/admitters/pod-eviction-admitter.go

```diff
@@ -92,14 +92,16 @@ func (admitter *PodEvictionAdmitter) Admit(ctx context.Context, ar *admissionv1.
 		markForEviction = true
 	}
 
+	// This message format is expected from descheduler.
```

Comment:
nit: maybe we can add some doc link regarding that requirement?

Reply:
https://github.com/kubevirt/community/blob/8c3dd8faae0247cb60c5bd67084af9869daf5f35/design-proposals/descheduler-support.md

Reply:
Thanks @orelmisan, but where does it say there that a descheduler requires this format?

Reply:
https://github.com/kubevirt/community/blob/8c3dd8faae0247cb60c5bd67084af9869daf5f35/design-proposals/descheduler-support.md?plain=1#L40

https://github.com/kubevirt/kubevirt/pull/11532

Reply:
Yeah, it doesn't specify this exact format. I also wonder where this comes from (is there a k8s link?).
In any case @xpivarc would be nice to link something IMO

Reply:
Please see https://github.com/kubevirt/kubevirt/commit/e65b39ce2f4f619bc04d53155ca00c56e50b7fc3 and https://github.com/kubernetes-sigs/descheduler/pull/1466